runtime/compiler/env/j9method.cpp

```diff
@@ -1116,7 +1116,7 @@ TR_ResolvedJ9MethodBase::isInlineable(TR::Compilation *comp)
 
 static intptrj_t getInitialCountForMethod(TR_ResolvedMethod *m, TR::Compilation *comp)
    {
-   TR::Options * options = comp->getOptions()->getCmdLineOptions();
+   TR::Options * options = comp->getOptions();
```

Comment:
I am torn about this one: we are asking about the initial count for method m which is not the same as the method being compiled. It's true that asking for the global options is not correct either because the user could have provided specific options for that method.

Reply:
Is that true 100% of the time? Are there cases where method m and method being compiled are the same method? 

```
TR_ResolvedMethod *compMethod = comp->getCurrentMethod();
count = compMethod->getInvocationCount();
initialCount = getInitialCountForMethod(compMethod, comp);
```
^ this looks like they are the same method, no?

Reply:
So this is the call where method m and method being compiled could be different
```
 intptrj_t initialCount = getInitialCountForMethod(this, comp);
```
is there a way to get `options` from the `this` pointer?